src/db/oracledb/jobs-dao.js

```diff
@@ -336,7 +308,7 @@ const isValidChangeReasonCode = async (connection, changeReasonCode) => {
  * @param {string} internalId Internal ID of a person
  * @returns {Error} returns error or null if no error occurred
  */
-const createOrUpdateJob = async (operation, osuId, body) => {
+const createOrUpdateJob = async (osuId, body) => {
```

Comment:
Not sure we should still call this function `createOrUpdate`. We don't have an operation parameter for this function anymore, and there is no explicit business logic here to show whether we are going to create or update jobs here (it is entirely handled at the contrib query by the job change reason codes now). Thoughts?

Reply:
I agree but I had trouble finding a better name. I changed it to `handleJob` thoughts?